pr-data.csv

```diff
@@ -126,17 +126,17 @@ https://github.com/adobe/asset-share-commons,ee3ef7051e3ea3eb7f5d904fac177bc5662
-https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.CheckoutTest.TestBacsDirectDebitDetailsSerialization,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681, A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library
-https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.CheckoutTest.TestBillDeskOnlineDetailsSerialization,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681, A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library
-https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.CheckoutTest.TestBilldeskWalletDetailsSerialization,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681, A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library
-https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.CheckoutTest.TestBlikDetailsSerialization,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681, A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library
-https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.CheckoutTest.TestDateSerializers,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681, A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library
-https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.CheckoutTest.TestDokuDetailsSerialization,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681, A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library
-https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.CheckoutTest.TestDotpayDetailsSerialization,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681, A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library
+https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.CheckoutTest.TestAchDetailsSerialization,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681," A previous PR (https://github.com/Adyen/adyen-java-api-library/pull/654) was opened but not accepted because of adding a library"
```

Comment:
"" (quotes added) for some strings

Reply:
That's weird. I'll remove the quotes, however, I didn't make these formatting changes, I think it might've been the command on the IDoFT repository to re-order the CSV file that might've made these changes.

Reply:
Okay, i've removed the formatting and committed and the csv file has passed the check!

Reply:
LGTM

Reply:
Thank you!